src/FissionErrors.ts

```diff
@@ -0,0 +1,73 @@
+export type ByteLength = number
+
+export interface AxiosError extends Error {
+    isAxiosError: boolean
+    config: {
+        headers: {
+            "Content-Length": ByteLength
+        }
+        maxContentLength: ByteLength
+    }
+}
+
+export class FileSizeError extends Error {
+  fileSizeBytes: ByteLength
+  maxFileSizeBytes: ByteLength
+  details: string
+  private axiosError: AxiosError
+  private AXIOS_DEFAULT_MAXIMUM = 10_000_000;
```

Comment:
This and all the other values that reference `10_000_000` bytes should be moved to the `constants.ts` file